docs/dns/rrsets.rst

```diff
@@ -213,6 +213,20 @@ https://www.huque.com/bin/gen_tlsa.  We are planning to provide a tool that is
 connected directly to our API in the future.  For full detail on how ``TLSA``
 records work, please refer to RFC 6698.
 
+The ``TLSA`` sha256 hash can also be generated with that command::
```

Comment:
_`that`_ normally refers to something previously mentioned. Instead use _`this`_ or _`the following`_.